BuildAddOn.py

```diff
@@ -271,46 +276,61 @@ def CopyResultToPackage (packageRootFolder, buildFolder, version, addOnName, pla
 
     elif platformName == 'MAC':
         subprocess.call ([
-            'cp', '-r',
+            'cp', '-R',
             sourceFolder / f'{addOnName}.bundle',
             packageFolder / f'{fileName}.bundle'
         ])
 
 
+def GetDevKitBuildNum (devKitData, version, platformName):
+    url = devKitData[platformName][version]
+    buildNum = url.split('/')[-2]
+    return buildNum
+
+
 # Zip packages
-def PackageAddOns (args, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):
+def PackageAddOns (args, devKitData, addOnName, platformName, acVersionList, languageList, buildFolder, packageRootFolder):

```

Comment:
The whole function should run only when `package` is set to true. I don't think that `allLocalizedVersions` should be checked inside the function.

Reply:
It is checked only to see whether Debug configured builds have been created or not.
Daily build results can also be packaged.

Reply:
Again, this script should not know about "Daily" builds. It's ok to package every kind of builds, but it still shouldn't check the `allLocalizedVersions` flag.